ydcommon/views.py

```diff
@@ -2,7 +2,11 @@
 from django.views.generic import TemplateView
 from django.conf import settings
 from django.contrib.admin.views.decorators import staff_member_required
-from django.core.urlresolvers import reverse
+try:
+    from django.urls import reverse
```

Comment:
Recall my comment in Demo project? This one should be done as file and imported from there, even in demo.